src/main/java/uk/gov/justice/digital/hmpps/whereabouts/model/WhereaboutsConfig.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.justice.digital.hmpps.whereabouts.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.*;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * Whereabouts Details
+ **/
+@SuppressWarnings("unused")
+@ApiModel(description = "Whereabouts Details")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+@ToString
+public class WhereaboutsConfig {
+
+    @NotNull
+    private boolean enabled;
```

Comment:
can't this just have a `@Data` annotation instead and then you won't need to specify the getters or setters (either that or a kotlin data class